packages/web-components/fast-components/src/breadcrumb-item/breadcrumb-item.styles.ts

```diff
@@ -0,0 +1,124 @@
+import { css } from "@microsoft/fast-element";
+import {
+    display,
+    focusVisible,
+    forcedColorsStylesheetBehavior,
+} from "@microsoft/fast-foundation";
+import {
+    accentForegroundActiveBehavior,
+    accentForegroundHoverBehavior,
+    accentForegroundRestBehavior,
+    neutralForegroundRestBehavior,
+    heightNumber,
+} from "../styles/index";
+import { SystemColors } from "@microsoft/fast-web-utilities";
+
+export const BreadcrumbItemStyles = css`
+    ${display("inline-flex")} :host {
+        background: transparent;
+        box-sizing: border-box;
+        font-family: var(--body-font);
+        font-size: var(--type-ramp-base-font-size);
+        fill: currentColor;
+        line-height: var(--type-ramp-base-line-height);
+        min-width: calc(${heightNumber} * 1px);
+        outline: none;
+    }
+
+    .listitem {
+        display: flex;
+        align-items: center;
+    }
+
+    .separator {
+        margin: 0 6px;
+    }
+
+    .control {
+        align-items: center;
```

Comment:
Perhaps it's the SVG's, but when I pulled this down (with the start/end fix mentioned here for "list" in breadcrumb) these don't look centered. Can you validate that start/end are actually being truly centered? Again, perhaps it's just visual with the SVG's.

Reply:
Yeah, it's the SVG, especially the "Home" SVG. I did some testing in `fast-anchor` and adding the same text and SVG, and it looks the same, which is slightly off-center. I think what is happening, is the SVG is aligning to the full height of the text and the underline. When the anchor is not being hovered, it looks off, but when you hovering on the anchor and the underline appears, it looks centered.
<img width="139" alt="no_underline" src="https://user-images.githubusercontent.com/37851220/92263504-95da4600-ee91-11ea-8ed5-eb5e74325b52.png">
<img width="147" alt="with_underline" src="https://user-images.githubusercontent.com/37851220/92263524-9d015400-ee91-11ea-85b0-da111cb4c036.png">